src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1997,6 +1997,11 @@ def _pick_fuzz_target():
   if not environment.is_engine_fuzzer_job():
     logs.log('Not engine fuzzer. Not picking fuzz target.')
     return None
+
+  if environment.get_value('FUZZ_TARGET_BUILD_BUCKET_PATH'):
+    logs.log('Not engine fuzzer. Not picking fuzz target.')
```

Comment:
Is this comment supposed to be the same as above?

Reply:
Nope. Fixed